content/freeze.md

```diff
@@ -55,7 +56,9 @@ No Freeze
 The **No Freeze** feature is a setting on an account that permanently gives up the ability to freeze counterparties. A business can use this feature to treat its issued funds as "more like physical money" in the sense that the business cannot interfere with customers trading it among themselves. The NoFreeze setting has two effects:
 
 * The issuing account can no longer use enable Individual Freeze on any counterparty.
+    [//]: # ("can no longer enable")
```

Comment:
Changed to "Can no longer enable Individual Freeze on trust lines to any counterparty."